tools/makeboards.py

```diff
@@ -225,6 +225,9 @@ def MakeBoardJSON(name, vendor_name, product_name, vid, pid, pwr, boarddefine, f
 # Raspberry Pi
 MakeBoard("rpipico", "Raspberry Pi", "Pico", "0x2e8a", "0x000a", 250, "RASPBERRY_PI_PICO", 2, "boot2_w25q080_2_padded_checksum")
 
+# DFRobot
```

Comment:
Please move this down after `#DeRuiLab` and before `#iLabs` to keep the listing alphabetical.